modules/library/metadata/src/main/java/org/geotools/math/Plane.java

```diff
@@ -44,86 +44,7 @@
  */
 public class Plane implements Cloneable, Serializable {
-    
-    /** Point3d data structure used to define Plane. */
-    public static class Point3d {
-        double x,y,z;
-        public Point3d( double x, double y, double z){
-            this.x=x;
-            this.y=y;
-            this.z=z;
-        }
-        public Point3d( double array[]) throws IllegalArgumentException {
-            if( array == null){
-                throw new NullPointerException("Vector array required");
-            }
-            if( array.length == 2 ){
-                this.x = array[0];
-                this.y = array[1];
-                this.z = Double.NaN;
-            }
-            else if( array.length == 3 ){
-                this.x = array[0];
-                this.y = array[1];
-                this.z = array[2];
-            }
-            else {
-                throw new IllegalArgumentException("Vector array (length " + array.length +") is not expected length of 3");
-            }
-        }
-        public Point3d( Point2D point ){
-            this.x = point.getX();
-            this.y = point.getY();
-            this.z = Double.NaN;
-        }
-        @Override
-        public int hashCode() {
-            final int prime = 31;
-            int result = 1;
-            long temp;
-            temp = Double.doubleToLongBits(x);
-            result = prime * result + (int) (temp ^ (temp >>> 32));
-            temp = Double.doubleToLongBits(y);
-            result = prime * result + (int) (temp ^ (temp >>> 32));
-            temp = Double.doubleToLongBits(z);
-            result = prime * result + (int) (temp ^ (temp >>> 32));
-            return result;
-        }
-        @Override
-        public boolean equals(Object obj) {
-            if (this == obj)
-                return true;
-            if (obj == null)
-                return false;
-            if (getClass() != obj.getClass())
-                return false;
-            Point3d other = (Point3d) obj;
-            if (Double.doubleToLongBits(x) != Double.doubleToLongBits(other.x))
-                return false;
-            if (Double.doubleToLongBits(y) != Double.doubleToLongBits(other.y))
-                return false;
-            if (Double.doubleToLongBits(z) != Double.doubleToLongBits(other.z))
-                return false;
-            return true;
-        }
-        @Override
-        public String toString() {
-            return "Point3d [x=" + x + ", y=" + y + ", z=" + z + "]";
-        }
-        /**
-         * Cartesian distance between two points.
-         * 
-         * @param pt second point
-         * @return cartesian distance between two points
-         */
-        public double distance(Point3d pt) {
-            double dX = Math.abs(this.x-pt.x);
-            double dY = Math.abs(this.y-pt.y);
-            double dZ = Math.abs(this.z-pt.z);
-            
-            return Math.sqrt( dX*dX + dY*dY + dZ*dZ);
-        }
-    }
+
```

Comment:
> The Point3D class are amassed without vertical spacing (coding convention breakage), none of public methods have javadocs. Unsure why the class is nested, the class seems general and it is being used without a Plane around

Parameter class was used to stage data for the constructor, data now provided using double[][] instead.